tox.ini

```diff
@@ -28,58 +28,6 @@ commands=
     pip freeze
     pytest --remote-data
 
-[testenv:s390x]
-# As of 2020-01-23, The s390x container on Travis has a bug where
-# /home/travis/.cache/pip/wheels is owned by root, which prevents
-# us from installing packages unless we disable caching.
-install_command= python -m pip install --no-cache-dir {opts} {packages}
-
-[testenv:prerelease]
-pip_pre= true
```

Comment:
pip_pre just installs any pre-release versions of our listed dependencies if they are available on PyPi. This is unnecessary because the devdeps jobs test using the development versions of our dependencies.

This is a hold over I missed when I crated devdeps.

Reply:
Thanks. Comparing some recent CI runs requirements-dev.txt covers all the packages that offer pre-releases at the moment.